gradle/plugins/src/main/kotlin/com/saveourtool/save/buildutils/publishing-configuration.gradle.kts

```diff
@@ -35,7 +35,12 @@ run {
 run {
     configurePublications()
 
-    if (hasProperty("signingKey")) {
-        configureSigning()
-    }
+    /*-
```

Comment:
why `save-api` and `test-analysis-core` have another publications? maybe migrate them to common one?

Reply:
`:save-cloud-common` is a multi-platform project, it just happens to have a default Maven publication readily available with no extra configuration.

Reply:
discussed offline, will leave it for future